WEEK_3/Week3_hw3/hw3.py

```diff
@@ -0,0 +1,13 @@
+def range_sequence(*args):
+    if len(args) == 1:
+        return range(args[0])
```

Comment:
Try running tests - they will fail with `TypeError: 'tuple' object cannot be interpreted as an integer`

You most likely want to use slices, 
`args[0][some_index:some_other_index:third_value]`